mysql-test/suite/tokudb/r/i_s_tokudb_lock_waits_released.result

```diff
@@ -68,9 +70,12 @@ TRX_ID	MYSQL_ID
 TRX_ID	MYSQL_ID
 commit;
 verify that the lock on the 1st transaction is released and replaced by the lock for the 2nd transaction
+Timeout in wait_condition.inc for select count(*)=1 from information_schema.tokudb_locks where locks_dname='./test/t-main'
```

Comment:
?

Reply:
This is actually a very good catch that I missed in my self-review.  This patch impacts locking and what locks are taken/needed for sure, but, in diagnosing this I discovered that I reverted too much and removed a needed sub/child FT transaction that needs to be used on a REPLACE INTO/IGNORE.  Pushed fixed code and test case with a new jenkins here https://ps.cd.percona.com/job/percona-server-5.7-param/62/